packages/amplify-category-interactions/src/provider-utils/awscloudformation/service-walkthroughs/lex-walkthrough.js

```diff
@@ -386,7 +386,7 @@ async function addSlot(context, intentName, resourceName, parameters) {
       slot.type = slot.type[0];
     }
 
-    slot.prompt = await prompter.prompt('Enter a prompt for your slot (e.g. What city?)', {
+    slot.prompt = await prompter.input('Enter a prompt for your slot (e.g. What city?)', {
```

Comment:
should we add test for this ?

Reply:
or convert to TS

Reply:
Like the idea of converting to TS, will do, from the initial try this file is a mess :)